tests/acm_observability/conftest.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import pytest
+import requests
+from const import HUB_CLUSTER
+from ocp_resources.multi_cluster_observability import MultiClusterObservability
+from ocp_resources.route import Route
+from pytest_testconfig import config as py_config
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.fixture(scope="session")
+def multi_cluster_observability(admin_client_scope_session):
+    observability_name = "observability"
```

Comment:
No need to save `observability_name`, you have `name` in the object

Reply:
the object is not yet initialized, is there a way to use the name of the object before its init?

Reply:
ok got your point

Reply:
done